GraphQL-TSC.md

```diff
@@ -126,9 +126,9 @@ The current voting members of the GraphQL TSC are:
 | [Benjie Gillam](https://github.com/benjie)         | Graphile              | Nov 1, 2020 | Dec 31, 2021 |
 | [Brielle Harrison](https://github.com/nyteshade)   | PayPal                | Nov 1, 2020 | Dec 31, 2021 |
 | [Dan Schafer](https://github.com/dschafer)         | Facebook              | Nov 1, 2020 | Dec 31, 2022 |
-| [Ivan Goncharov](https://github.com/IvanGoncharov) | Self                  | Nov 1, 2020 | Dec 31, 2021 |
+| [Ivan Goncharov](https://github.com/IvanGoncharov) | Apollo                | Nov 1, 2020 | Dec 31, 2021 |
 | [Matt Mahoney](https://github.com/mjmahone)        | Facebook              | Nov 1, 2020 | Dec 31, 2021 |
-| [James Baxley](https://github.com/jbaxleyiii)      | Apollo GraphQL        | Nov 1, 2020 | Dec 31, 2021 |
+| [James Baxley](https://github.com/jbaxleyiii)      | ex-Apollo             | Nov 1, 2020 | Dec 31, 2021 |
```

Comment:
@jbaxleyiii I changed your `Affiliation` column to "ex-Apollo" so it doesn't look as if Apollo is overrepresented in this group.
Is it ok for you?
If you want it to be something else please just comment here and I will change it.

Reply:
@IvanGoncharov I just talked to James; would you mind changing it to "Carbon Health"?

Reply:
@hwillson Done 👍